samples/BlazorClientSide/wwwroot/index.html

```diff
@@ -7,10 +7,12 @@
     <base href="/" />
     <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
     <link href="css/site.css" rel="stylesheet" />
+    <link href="_content/Blazored.Typeahead/blazored-typeahead.css" rel="stylesheet" />
```

Comment:
Could we move this above the site.css? This shows the better practice of library styles coming before site-specific styles.